src/interpreter.ts

```diff
@@ -217,7 +240,26 @@ export default class SummaryBallotInterpreter implements Interpreter {
       return bmdResult
     }
 
-    return await this.interpretHMPBFile(election, ballotImageData)
+    try {
+      const hmpbResult = await this.interpretHMPBFile(election, ballotImageData)
+
+      if (hmpbResult) {
+        return hmpbResult
+      }
+    } catch {
+      // ignore for now
+    }
+
+    try {
+      const metadata = metadataFromBytes(ballotImageData.qrcode)
```

Comment:
QR code is not extracted first?

Reply:
The QR code is extracted first. This is using the information already decided from the QR code to at least attempt to interpret it as metadata for a HMPB.